package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/fastify/fastify-sensible#readme",
   "devDependencies": {
-    "fastify": "^1.13.2",
+    "fastify": "^2.0.0-rc.2",
```

Comment:
I guess you can directly use `^2.0.0`, since the npm [semver](https://www.npmjs.com/package/semver) parser does not support tags such as `rc.x`.

Reply:
^2.0.0 does not pick up ^2.0.0-rc.2